packages/core/src/integrations/inboundfilters.ts

```diff
@@ -36,198 +35,184 @@ export class InboundFilters implements Integration {
-      return false;
-    }
+/** JSDoc */
+function _isIgnoredError(event: Event, ignoreErrors: Partial<InboundFiltersOptions>['ignoreErrors']): boolean {
+  if (!ignoreErrors || !ignoreErrors.length) {
+    return false;
+  }
 
-    try {
-      // @ts-ignore can't be a sentry error if undefined
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
-      return event.exception.values[0].type === 'SentryError';
-    } catch (e) {
-      // ignore
-    }
+  return _getPossibleEventMessages(event).some(message =>
+    ignoreErrors.some(pattern => isMatchingPattern(message, pattern)),
+  );
+}
 
+/** JSDoc */
+function _isDeniedUrl(event: Event, denyUrls: Partial<InboundFiltersOptions>['denyUrls']): boolean {
+  // TODO: Use Glob instead?
+  if (!denyUrls || !denyUrls.length) {
     return false;
   }
+  const url = _getEventFilterUrl(event);
+  return !url ? false : denyUrls.some(pattern => isMatchingPattern(url, pattern));
+}
 
-  /** JSDoc */
-  private _isIgnoredError(event: Event, options: Partial<InboundFiltersOptions>): boolean {
-    if (!options.ignoreErrors || !options.ignoreErrors.length) {
-      return false;
-    }
-
-    return this._getPossibleEventMessages(event).some(message =>
-      // Not sure why TypeScript complains here...
-      (options.ignoreErrors as Array<RegExp | string>).some(pattern => isMatchingPattern(message, pattern)),
-    );
+/** JSDoc */
+function _isAllowedUrl(event: Event, allowUrls: Partial<InboundFiltersOptions>['allowUrls']): boolean {
+  // TODO: Use Glob instead?
+  if (!allowUrls || !allowUrls.length) {
+    return true;
   }
+  const url = _getEventFilterUrl(event);
+  return !url ? true : allowUrls.some(pattern => isMatchingPattern(url, pattern));
+}
 
-  /** JSDoc */
-  private _isDeniedUrl(event: Event, options: Partial<InboundFiltersOptions>): boolean {
-    // TODO: Use Glob instead?
-    if (!options.denyUrls || !options.denyUrls.length) {
-      return false;
+/** JSDoc */
+function _getPossibleEventMessages(event: Event): string[] {
+  if (event.message) {
+    return [event.message];
+  }
+  if (event.exception) {
+    try {
+      const { type = '', value = '' } = (event.exception.values && event.exception.values[0]) || {};
+      return [`${value}`, `${type}: ${value}`];
+    } catch (oO) {
+      if (isDebugBuild()) {
+        logger.error(`Cannot extract message for event ${getEventDescription(event)}`);
+      }
+      return [];
     }
-    const url = this._getEventFilterUrl(event);
-    return !url ? false : options.denyUrls.some(pattern => isMatchingPattern(url, pattern));
   }
+  return [];
+}
 
-  /** JSDoc */
-  private _isAllowedUrl(event: Event, options: Partial<InboundFiltersOptions>): boolean {
-    // TODO: Use Glob instead?
-    if (!options.allowUrls || !options.allowUrls.length) {
-      return true;
+/** JSDoc */
+function _isSentryError(event: Event, ignoreInternal: Partial<InboundFiltersOptions>['ignoreInternal']): boolean {
```

Comment:
```suggestion
function _isSentryError(event: Event, ignoreInternal: boolean): boolean {
```

It's not going to change, so why not just say what it is?

Reply:
> It's not going to change, so why not just say what it is?

It's to communicate intent - the whole purpose of aliasing types.